ctapipe/instrument/tests/test_trigger.py

```diff
@@ -102,3 +115,118 @@ def test_software_trigger_simtel(allowed_tels):
         for e, expected_tels in zip(source, expected):
             trigger(e)
             assert_equal(e.trigger.tels_with_trigger, expected_tels)
+            assert_all_tel_keys(e, expected_tels)
+
+
+def test_software_trigger_simtel_single_lsts():
+    from ctapipe.instrument.trigger import SoftwareTrigger
+
+    path = "dataset://gamma_divergent_LaPalma_baseline_20Zd_180Az_prod3_test.simtel.gz"
+
+    # remove 3 LSTs, so that we trigger the 1-LST condition
+    allowed_tels = [1, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]
+    expected = [
+        [12, 16],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [5, 6, 7, 12, 15, 16, 17, 18],
+        [13, 14],
+        [],
+        [7, 12],
+        [5, 17],
+        [13, 19],
+        [],
+        [],
+        [5, 11, 18],
+        [17, 18],
+        [7, 12],
+        [],
+    ]
+
+    with EventSource(
+        path, focal_length_choice="EQUIVALENT", allowed_tels=allowed_tels
+    ) as source:
+
+        trigger = SoftwareTrigger(
+            subarray=source.subarray,
+            min_telescopes=2,
+            min_telescopes_of_type=[
+                ("type", "*", 0),
+                ("type", "LST*", 2),
+            ],
+        )
+
+        for e, expected_tels in zip(source, expected):
+            print(e.trigger.tels_with_trigger)
+            trigger(e)
+            print(e.trigger.tels_with_trigger, expected_tels)
+            assert_equal(e.trigger.tels_with_trigger, expected_tels)
+            assert_all_tel_keys(e, expected_tels)
+
+
+def test_software_trigger_simtel_process(tmp_path):
```

Comment:
Should probably be `test_process_software_trigger_simtel` to be consistent with the naming of the other tests outside of `test_process.py` that invokes the full `ctapipe-process` machinery.

I also wonder if it isn't more appropriate to put the generation of the two files into `conftest.py` since that place  is already full of `run_tool(ProcessTool)` invocations, whereas they don't seem to happen elsewhere except inside `tool/tests/` (where you expect it to show up).

Reply:
conftest is meant for fixtures that are used by multiple tests. Here we actually just want to look at these two specific files and compare them.

Reply:
Sure, I guess we can just move it once we invent a second test that could use one of these files.